android_client/app/src/main/java/com/microsoft/hydralab/android/client/MainActivity.java

```diff
@@ -54,14 +55,17 @@
 
 public class MainActivity extends Activity {
     private static final String TAG = "ScreenRecorder";
+    private static final String Default_ACV_PROFILE = "AVCProfileBaseline-AVCLevel52";
```

Comment:
Why not use the “Default” value?
```
private static final String Default_ACV_PROFILE = "Default"
```

I tested the above code on my phone and it worked well.

Reply:
Last time I try this one caused crash. I'll try again with latest change

Reply:
Still got a crash but resolving with adding another check.